lab_5_scrapper/scrapper.py

```diff
@@ -26,11 +83,52 @@ def _extract_config_content(self) -> ConfigDTO:
         Returns:
             ConfigDTO: Config values
         """
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            config = json.load(file)
+        return ConfigDTO(
```

Comment:
how would you do it, if I asked you to make instance creation with arguments extraction from dictionary?

Reply:
Maybe I changed it

Reply:
excellent 🥇